### ISSUE TITLE
May i ask if there is a way to use dcm image to train the model? If its available, what should i do.

### ISSUE THREAD

**Issue by Ryanye2000** [SysCV/sam-hq#97]
May i ask if there is a way to use dcm image to train the model? 
If its available, what should i do.
Do i have to modify dataloader.py or something?

**Comment by lkeab** [comment]
Yes, you need to replace our HQ-Seg-44K training data into your own and modified the dataloader accordingly.

**Comment by Ryanye2000** [comment]
OK I will try, thx a lot
